thunderbird/processes/wps_generate_climos.py

```diff
@@ -129,32 +130,17 @@ def __init__(self):
         )
 
     def dry_run_info(self, filepath, climo):
-        output = ["Dry Run"]
-        output.append(f"File: {filepath}")
-        try:
-            input_file = CFDataset(filepath)
-        except Exception as e:
-            output.append(f"{e.__class__.__name__}: {e}")
-        else:
-            periods = input_file.climo_periods.keys() & climo
-            output.append(f"climo_periods: {periods}")
-            for attr in "project institution model emissions run".split():
-                try:
-                    output.append(f"{attr}: {getattr(input_file.metadata, attr)}")
-                except Exception as e:
-                    output.append(f"{attr}: {e.__class__.__name__}: {e}")
-            output.append(f"dependent_varnames: {input_file.dependent_varnames()}")
-            for attr in "time_resolution is_multi_year_mean".split():
-                output.append(f"{attr}: {getattr(input_file, attr)}")
-
         filename = os.path.basename(filepath).split(".")[
             0
         ]  # Uniquely identify each dry run file
         filename += "_dry.txt"
         filename = os.path.join(self.workdir, filename)
         with open(filename, "w") as f:
-            for line in output:
-                f.write(f"{line}\n")
+            f.write("Dry Run\n")
+            logging.basicConfig(filename=filename, level=logging.INFO)
+            dry_run_handler(filepath, climo)
+            for handler in logging.root.handlers[:]:
+                logging.root.removeHandler(handler)
```

Comment:
So as I understand it, we are logging to the `dry.txt` file here through the `python logging` module. I think this is fairly messy and could be done in a better way. In fact, we do this in a better way in the [`generate_prsn` module](https://github.com/pacificclimate/thunderbird/blob/master/thunderbird/processes/wps_generate_prsn.py#L153). We use a [`dry` method](https://github.com/pacificclimate/climate-explorer-data-prep/blob/master/dp/generate_prsn.py#L22) that has a flexible output. I think we need to double back to `ce-dataprep` and adjust the `dry` method. It was an oversight on my part not to see the issue ahead of time, my bad.

Here is the new [issue](https://github.com/pacificclimate/climate-explorer-data-prep/issues/135).